core/rs/core/src/c.rs

```diff
@@ -640,3 +636,32 @@ fn bindgen_test_layout_crsql_ExtData() {
         )
     );
 }
+
+pub trait CPointer<T> {
+    fn into_c_ptr(self) -> *mut T;
```

Comment:
add a doc comment stating that the memory is no longer tracked by Rust and whomever was passed the pointer is responsible for freeing the memory.